modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,146 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk HEC.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify which business groups allow content to be exported to connections. You can also edit and delete connections and configurations. 
+
+include::partial$telemetry-exporter-notes.adoc[tag=hourDelay]
+
+// add back to configs: "configuration to specify the types of data Anypoint Platform exports as well as..."
+
+== Supported Third-Party Apps
+
+The Telemetry Exporter feature enables you to create connections with most third-party observability and analytics apps that have OpenTelemetry-supported endpoints and allow you to create an instrumentation key or access token. Azure Monitor and Splunk HEC are also compatible. 
+
+== Before You Begin
+
+Before you configure the Telemetry Exporter feature, ensure that you have:
+
+* To create or modify connections, an Anypoint Platform account with the Telemetry Exporter Administrator permission
+* To create or modify configurations, an Anypoint Platform account with the Telemetry Exporter Configuration Manager permission
+* Access information for your third-party app, including the OpenTelemetry-supported endpoint. For example, the endpoint URL for Azure Monitor is `https://dc.services.visualstudio.com/v2/track`.
+* The ability to create an instrumentation key or access token for your third-party app
+
+== Access Telemetry Exporter
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=eitherPermission]
+
+
+You can switch between the *Configurations* and *Connections* tabs to create, edit, or view data configurations and connections to third-party apps.
+
+== Manage a Connection 
+
+Anypoint Platform needs permission to send data to third-party apps. Create a _connection_ with a third-party app, such as New Relic or Splunk, to authorize data transfers. You can configure up to 10 connections.
+
+You need the Telemetry Exporter Administrator permission to manage connections.
+
+include::partial$telemetry-exporter-notes.adoc[tag=hourDelay]
+
+=== Create a Connection
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=connectionPermission]
+
+. Click the *Connections* tab.
+. Click *New Connection*.
++
+The *Create New Connection* interface appears, showing the *Connection Details* page.
+. In the *Connection Details* page, enter the following information: 
+* *Connection Name*: A name for your connection using only alphanumeric characters and hyphens.
```

Comment:
Kim always had us use this format when we described a field. It makes the text more skimmable. 

Connection Name
+
Name for your connection, using only 
alphanumeric characters and hyphens.

You can also use a table.